### PR TITLE
fix(build.sh): add required if block end marker

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

**Comment by arschles** [comment]
this is untested in production

**Comment by arschles** [comment]
removing the `v2-beta` milestone since this isn't critical for the release

**Comment by jchauncey** [comment]
we should clear the LGTMs on it as well

**Comment by arschles** [comment]
cleared

**Comment by smothiki** [comment]
I dont see any missing IFs in the code

File: rootfs/builder/build.sh [hunk 1]
```diff
@@ -206,4 +206,5 @@ if [[ "$slug_file" != "-" ]]; then
 				curl -0 -s -o /dev/null -X PUT -T $slug_file "$put_url"
 			fi
 		fi
+	fi
```
Comment (bacongobbler) [inline rootfs/builder/build.sh:209]: I think this is an indentation issue. Notice that there is 4 `if` levels here and 5 `fi` levels.

**Comment by mboersma** [comment]
I would expect `shellcheck` to complain about this. But maybe #54 was merged after this was created?

@arschles could you rebase this so we can see it get linted in Travis CI?

**Comment by mboersma** [comment]
Never mind, I just clicked the re-run button in Travis and `shellcheck` agrees with @smothiki:

``` console
Unable to find image 'quay.io/deis/go-dev:0.11.0' locally
0.11.0: Pulling from deis/go-dev
Status: Downloaded newer image for quay.io/deis/go-dev:0.11.0
In rootfs/builder/build.sh line 192:
fi
^-- SC1089: Parsing stopped here. Is this keyword correctly matched up?
make: *** [test-style] Error 1
```

**Comment by arschles** [comment]
ah, ok. I think I can close this then

**Pull request closed without merging**